### PR TITLE
Clarifies installing packages locally on Windows

### PR DESCRIPTION
**Pull request by ecowden**
For #421 - 

This is my first PR here, so I tried to grab a simple one. Feedback welcome!

### CODE REVIEW

**Comment by ashleygwilliams** [comment]
yay! this is great. thanks so much :dolphin: :space_invader: :100:

File: content/getting-started/installing-npm-packages-locally.md [hunk 1]
```diff
@@ -21,11 +21,11 @@ npm install lodash
 
 This will create the `node_modules` directory (if one doesn't exist yet), and will download the package to that directory.
 
-Test: Run `ls node_modules`. You should see a lodash directory inside.
+Test: Run `ls node_modules` on Unix, or `dir node_modules` on Windows. You should see a lodash directory inside.
```
Comment (ashleygwilliams) [inline content/getting-started/installing-npm-packages-locally.md]: could you update this to say "*nix systems, e.g. OSX"

**Comment by ashleygwilliams** [comment]
just a little nit and then i'll merge!

**Comment by ecowden** [comment]
Oh, sure! I want to get this right. :sunglasses: 

How's this?

**Comment by ashleygwilliams** [comment]
:+1:

**Pull request merged**